test/jdk/java/awt/color/ICC_ProfileSetNullDataTest.java

```diff
@@ -23,27 +23,32 @@
 
 import java.awt.color.ColorSpace;
 import java.awt.color.ICC_Profile;
+import java.util.HashMap;
+import java.util.Map;
 
 /**
  * @test
  * @bug 4823896 7042594
  * @summary Test checks behavior of the ICC_Profile.setData(int, byte[])
  */
 public final class ICC_ProfileSetNullDataTest {
-    private static final int[] colorSpace = {
-            ColorSpace.CS_sRGB, ColorSpace.CS_LINEAR_RGB,
-            ColorSpace.CS_PYCC, ColorSpace.CS_GRAY,
-            ColorSpace.CS_CIEXYZ
-    };
+    private static final HashMap<Integer, String> colorSpace = new HashMap<>(Map.of(
+            ColorSpace.CS_sRGB, "CS_sRGB",
+            ColorSpace.CS_PYCC, "CS_PYCC",
+            ColorSpace.CS_GRAY, "CS_GRAY",
+            ColorSpace.CS_CIEXYZ, "CS_CIEXYZ",
+            ColorSpace.CS_LINEAR_RGB, "CS_LINEAR_RGB"
+    ));
 
     public static void main(String[] args) {
         byte[] tagData = null;
-        for (int cs : colorSpace) {
+        for (int cs : colorSpace.keySet()) {
             ICC_Profile builtInProfile = ICC_Profile.getInstance(cs);
             ICC_Profile profile = ICC_Profile.getInstance(builtInProfile.getData());
             try {
                 profile.setData(ICC_Profile.icSigCmykData, tagData);
```

Comment:
```suggestion
                profile.setData(ICC_Profile.icSigCmykData, null);
```
Should this be simplified?